packages/view/backend/sirius-components-view-diagram-customnodes-edit/src/main/java/org/eclipse/sirius/components/view/diagram/customnodes/provider/CustomNodesEditPlugin.java

```diff
@@ -95,8 +63,7 @@ protected Object doGetImage(String key) throws IOException {
      * @return the singleton instance.
      * @generated
      */
-    @Override
-    public ResourceLocator getPluginResourceLocator() {
+    public static Implementation getPlugin() {
```

Comment:
Aren't we losing the custom code needed to support SVG icons here?

Reply:
It seems that we may have been missing a generated not somewhere. I'll restore the previous code

Reply:
Done